### PR TITLE
Add missing parachain types

### PR DESCRIPTION
**Pull request by joelamouche**
solves https://github.com/polkadot-js/apps/issues/4217

### CODE REVIEW

File: packages/types/src/interfaces/parachains/definitions.ts [hunk 1]
```diff
@@ -155,6 +155,13 @@ export default {
     ParaScheduling: {
       _enum: ['Always', 'Dynamic']
     },
+    PersistedValidationData:{
+      parent_head: 'HeadData',
+      block_number: 'BlockNumber',
+      hrmp_mqc_heads: 'Vec<(Id, Hash)>',
+      dmq_mqc_head: 'Hash',
+      max_pov_size: 'u32',
+    },
```
Comment (jacogr) [inline packages/types/src/interfaces/parachains/definitions.ts:164]:
```suggestion
    PersistedValidationData:{
      parentHead: 'HeadData',
      blockNumber: 'BlockNumber',
      hrmpMqcHeads: 'Vec<(Id, Hash)>',
      dmqMqcHead: 'Hash',
      maxPovSize: 'u32',
    },
```

File: packages/types/src/interfaces/parachains/definitions.ts [hunk 2]
```diff
@@ -188,13 +195,24 @@ export default {
       }
     },
     SubId: 'u32',
+    TransientValidationData:{
+      max_code_size: 'u32',
+      max_head_data_size: 'u32',
+      balance: 'Balance',
+      code_upgrade_allowed: 'Option<BlockNumber>',
+      dmq_length: 'u32',
+    },
```
Comment (jacogr) [inline packages/types/src/interfaces/parachains/definitions.ts:204]:
```suggestion
    TransientValidationData:{
      maxCodeSize: 'u32',
      maxHeadDataSize: 'u32',
      balance: 'Balance',
      codeUpgrade_allowed: 'Option<BlockNumber>',
      dmqLength: 'u32',
    },
```

**Comment by jacogr** [comment]
`yarn lint` locally will/should also fix these. (The `--fix` is not applied on CI since it is very dangerous, especially with types especially when stuff gets moved alphabetically in types due to some missing override...)

**Comment by joelamouche** [comment]
updated with lint

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**